templates/web/cyclinguk/email_sent.html

```diff
@@ -0,0 +1,31 @@
+[% pre_container_extra = BLOCK %]
+<div class="bg-cyclinguk">
+    <div class="confirmation-header confirmation-header--inbox">
+
+        <h1>[% loc("Nearly done! Now check your email&hellip;") %]</h1>
+    
+        <p>
+        [% IF email_type == 'problem' %]
+            Click the link in our confirmation email to publish this problem.
+        [% ELSIF email_type == 'update' %]
+            [% loc("Click the link in our confirmation email to publish your update.") %]
+        [% ELSIF email_type == 'alert' %]
+            [% loc("Click the link in our confirmation email to activate your alert.") %]
+        [% END %]
+        </p>
+    
+        [% TRY %][% INCLUDE '_email_sent_extra.html' %][% CATCH file %][% END %]
+    
+        <p>
+            [% loc("Can&rsquo;t find our email? Check your spam folder&nbsp;&ndash; that&rsquo;s the solution 99% of the time.") %]
+        </p>
+    
+    </div>
+</div>
+[% END %]
+
+[% INCLUDE 'header.html', bodyclass = 'fullwidthpage', title = loc('Create a report') %]
+
+[% TRY %][% INCLUDE 'emails/_emails_content.html' %][% CATCH file %][% END %]
```

Comment:
We deliberately do not include anything else on the check your email page so as not to distract people when the thing they *have* to do now is check their email and confirm their thing. So we can add these things here, but I'm not sure we should. Looking at the page, I just completely gloss over the important message of checking your email, assuming it's some kind of banner message I can ignore
![image](https://github.com/mysociety/fixmystreet/assets/154364/4be130f0-970d-4d3a-83b8-1d82b3cdf49c)

Reply:
Sounds good and it makes sense. So What I've done now is display the content below on:

- 404
- Confirm Alert
- Confirm Problem
- Confirm Update

All the errors or anything that requires the user to execute an action shouldn't have that. I might need to add something to those ones, so the hero and footer doesn't clash.